tests/installation/addon_products_sle.pm

```diff
@@ -149,48 +152,52 @@ sub run {
         advance_installer_window('inst-addon');
         set_var('SKIP_INSTALLER_SCREEN', 0);
     }
-    if ($self->process_unsigned_files([qw(inst-addon addon-products)])) {
-        assert_screen_with_soft_timeout(
-            [qw(inst-addon addon-products)],
-            timeout      => 60,
-            soft_timeout => 30,
-            bugref       => 'bsc#1123963');
+    unless (check_var('FLAVOR', 'Full')) {
```

Comment:
Please, put a comment explaining why we don't need this for full medium. I also think that we should make this part work for the new medium, but we can address this later.

Reply:
b6b3e0280